features/search.feature

```diff
@@ -5,18 +5,28 @@ Feature: Search for a stock
 
   Scenario: Search for an invalid stock
     Given I am on the homepage
-    And my cursor is in the searchbar
+    And I click on the searchbar
     When I type text that is an invalid stock symbol
     And I click the search icon
     Then I should see an error message that the stock was not found
 
-  # TODO: add scenario for valid stock search
+  Scenario: Search for a valid stock
+    Given I am on the homepage
+    And I click on the searchbar
+    When I type text that is a valid stock symbol
+    And I click the search icon
+    Then I should see some stock results
 
-  # TODO: add scenario for search with ENTER
+  Scenario: Search for a stock with Enter key
+    Given I am on the homepage
+    And I click on the searchbar
+    When I type text that is an invalid stock symbol
```

Comment:
I prefer the idea of searching for a valid stock here.  A couple reasons:
- The title is "Search for a stock with Enter key" but an invalid stock symbol isn't actually a stock... by definition. 
- The "normal" case/happy path is probably to search for a valid stock, and I believe it's good to only change one variable at a time per test.  In this case the variable is the search submission method: ENTER vs click button